Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/EventCheckIn.kt

```diff
@@ -6,6 +6,16 @@ interface EventCheckIn {
 
     val id: Long
     val guid: String
-    val startTime: Instant
-    val endTime: Instant
+    val version: Int
+    val type: Int
+    val description: String
+    val address: String
+    val traceLocationStart: Instant
```

Comment:
`Start` and `End` can be nullable according to the TechSpec

Reply:
fixed it!